### PR TITLE
Add Windowing snippets for Python SDK.

### PR DESCRIPTION
**Pull request by charlesccychen**

### CODE REVIEW

**Comment by charlesccychen** [comment]
R: @aaltay

File: sdks/python/apache_beam/examples/snippets/snippets_test.py [hunk 1]
```diff
@@ -775,6 +775,112 @@ def test_count(self):
     perkey_counts = occurrences | snippets.Count()
     self.assertEqual({('cat', 3), ('dog', 2)}, set(perkey_counts))
 
+  def test_setting_fixed_windows(self):
+    p = TestPipeline()
+    unkeyed_items = p | 'create' >> beam.Create([22, 33, 55, 100, 115, 120])
```
Comment (aaltay) [inline sdks/python/apache_beam/examples/snippets/snippets_test.py]: Should we remove the `create` label? the transform is already labeled `Create`.

File: sdks/python/apache_beam/examples/snippets/snippets_test.py [hunk 2]
```diff
@@ -775,6 +775,112 @@ def test_count(self):
...
+    # [END setting_global_window]
+    summed = (session_windowed_items
+              | 'group' >> beam.GroupByKey()
+              | 'combine' >> beam.CombineValues(sum))
+    unkeyed = summed | 'unkey' >> beam.Map(lambda x: x[1])
+    beam.assert_that(unkeyed, beam.equal_to([56]))
+    p.run()
+
+  def test_setting_timestamp(self):
+
```
Comment (aaltay) [inline sdks/python/apache_beam/examples/snippets/snippets_test.py]: nit: remove empty line.

**Comment by charlesccychen** [comment]
Thanks, PTAL.

**Comment by aaltay** [comment]
LGTM, thank you.

**Pull request closed without merging**